cdap-watchdog/src/main/java/io/cdap/cdap/logging/plugins/RollingLocationLogAppender.java

```diff
@@ -110,7 +122,11 @@ public void doAppend(ILoggingEvent eventObject) throws LogbackException {
                                                                                            .getMDCPropertyMap());
         rollover(logLocationIdentifier, eventObject);
         OutputStream locationOutputStream = locationManager.getLocationOutputStream(logLocationIdentifier, filePath);
-        setOutputStream(locationOutputStream);
+
+        //change the locationOutputStream to which outputStream in the OutputStreamAppender points
```

Comment:
It's better to override `append` and not `doAppend`. This way you can drop tons of copy-paste from the parent

Reply:
Done